Templates/SaaSFileTrigger-Batch/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "SaaSFileTrigger - Batch",
+    "description": "A Batch function that will be run whenever a file is added to a SaaS provider.",
```

Comment:
That issue doesn't really seem to be about what I'm asking here. We can take this one offline - I only mentioned it here because I just thought of it.